<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kontakti</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ime</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leIme">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Priimek</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePriimek">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTelefon">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefon</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Naslov</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNaslov">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btShrani">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>49</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Shrani</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btIsci">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>19</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Išči</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btPobrisi">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>79</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Pobriši</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btSprazni">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>109</y>
     <width>75</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Sprazni</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
